### PR TITLE
Add `pointerType` description to docs.

### PR DESCRIPTION
**Pull request by m-bert**
## Description

This PR adds information about `pointerType` field in event to our documentation.

## Test plan

Run docs.

### CODE REVIEW

File: docs/docs/gestures/_shared/base-gesture-event-data.md [hunk 1]
```diff
@@ -7,3 +7,11 @@ Current [state](/docs/fundamentals/states-events) of the handler. Expressed as o
 ### `numberOfPointers`
 
 Represents the number of pointers (fingers) currently placed on the screen.
+
+### `pointerType`
+Indicates the type of pointer device being used. This value is represented by one of the constants defined in the `PointerType` enum, which includes the following fields:
```
Comment (kacperkapusciak) [inline docs/docs/gestures/_shared/base-gesture-event-data.md:12]:
```suggestion
Indicates the type of pointer device in use. This value is represented by the `PointerType` enum, which includes the following fields:
```

You should strongly prefer writing technical documentation in an **active voice**. More info: https://developers.google.com/tech-writing/one/active-voice

The second sentence is a bit harder to write in active voice so I made it shorter.

File: docs/docs/gestures/_shared/base-gesture-event-data.md [hunk 2]
```diff
@@ -7,3 +7,11 @@ Current [state](/docs/fundamentals/states-events) of the handler. Expressed as o
...
+
+- `TOUCH`
+- `STYLUS`
+- `MOUSE`
+- `OTHER`
```
Comment (kacperkapusciak) [inline docs/docs/gestures/_shared/base-gesture-event-data.md:17]: The first three are self-explanatory but I really would like to learn (and probably developers reading this documentation) what `OTHER` is.

File: docs/docs/gestures/_shared/base-gesture-event-data.md [hunk 3]
```diff
@@ -7,3 +7,11 @@ Current [state](/docs/fundamentals/states-events) of the handler. Expressed as o
 ### `numberOfPointers`
 
 Represents the number of pointers (fingers) currently placed on the screen.
+
+### `pointerType`
```
[the remark this answers is not in the document]
Comment (m-bert) [reply docs/docs/gestures/_shared/base-gesture-event-data.md:11]: Done in [1cf8899](https://github.com/software-mansion/react-native-gesture-handler/pull/2791/commits/1cf88999ea581990bbaad6c2988def4c2f223075)

File: docs/docs/gestures/_shared/base-gesture-event-data.md (hunk 2 above)
Comment (m-bert) [reply docs/docs/gestures/_shared/base-gesture-event-data.md]: Done in [3a53822](https://github.com/software-mansion/react-native-gesture-handler/pull/2791/commits/3a538226389517f601c5658c85f9f542cd543607), let me know what you think!

**Review by kacperkapusciak — APPROVED** [review]
:shipit:

**Pull request merged**